src/Phinx/Db/Adapter/SqlServerAdapter.php

```diff
@@ -44,6 +44,12 @@ class SqlServerAdapter extends PdoAdapter implements AdapterInterface
 
     protected $signedColumnTypes = ['integer' => true, 'biginteger' => true, 'float' => true, 'decimal' => true];
 
+    const INT_TINY = 255;
+    const INT_SMALL = 65535;
+    const INT_MEDIUM = 16777215;
+    const INT_REGULAR = 4294967295;
+    const INT_BIG = 18446744073709551615;
```

Comment:
Is is possible to declare those constants as strings? Sorry if I am wrong, but I guess INT_BIG would overflow in 32bit PHP, and if you want to use unsigned bigint, it would overflow even in 64bit PHP.